alonet/detr/coco_data_modules.py

```diff
@@ -157,6 +157,7 @@ def val_check(self):
         )
         self.sample = self.coco_val.sample or self.sample  # Update sample if user prompt is given
         self.CATEGORIES = self.coco_val.CATEGORIES if hasattr(self.coco_val, "CATEGORIES") else None
+        self.labels_names = self.coco_val.labels_names if hasattr(self.coco_val, "labels_names") else None
```

Comment:
what is the difference between CATEGORIES & labels_names ?

Reply:
`CATEGORIES` is a set, and `label_names` is a list. For that, `CATEGORIES` keep only the unique values. But you are right. This attribute is not as important as `labels_names`. I will remove it